seeds/index.js

```diff
@@ -33,6 +40,24 @@ const seedAll = async () => {
   await seedUser();
   console.log('\n----- USER SEEDED -----\n');
 
+  await seedApi();
+  console.log('\n----- API SEEDED -----\n');
+
+  await seedCardCategory();
+  console.log('\n----- CARD CATEGORY SEEDED -----\n');
+
+  await seedCategory();
```

Comment:
Category needs to be populated before cardCategory since it needs the ID. I would recommend just putting cardCategory as the last seed